### ISSUE TITLE
BW -> color photos details

### ISSUE THREAD

**Issue by lambdaofgod** [junyanz/pytorch-CycleGAN-and-pix2pix#306]
In the paper there is a mention of the colorizing model. What are the details?
Specifically I'm interested in:
- hyperparameters, especially learning rates, lambda_1 and learning rate decay
- what were the sizes of kernels in generator and discriminator

**Comment by junyanz** [comment]
See this [post](https://github.com/phillipi/pix2pix/issues/145)
